Source/WebKit/WebProcess/Plugins/PDF/PDFIncrementalLoader.mm

```diff
@@ -397,8 +399,8 @@
 
     for (auto identifier : handledRequests) {
         auto request = m_requestData->outstandingByteRangeRequests.take(identifier);
-        if (request.streamLoader())
-            cancelAndForgetStreamLoader(*request.streamLoader());
+        if (RefPtr streamLoader = request.protectedStreamLoader())
```

Comment:
I think our convention is to call the regular non-protected version of `request.streamLoader` here and store it in RefPtr.

Reply:
Ah, I see. Still I do too much.